File: 7KYU/Basic Calculator/solution.py
def calculate(num1, operation, num2): 
    if operation == '+':
        return num1 + num2
    elif operation == '-':
        return num1 - num2
    elif operation == '*':
        return num1 * num2
    elif operation == '/' and num2 != 0:
        return num1 / num2
    return None

File: 7KYU/Basic Calculator/solution.cs
using System;

public static class Calculator
{
  public static double Execute(double num1, char op, double num2)
  {
    if(op == '+')
      return num1 + num2;
    if(op == '-')
      return num1 - num2;
    if(op == '/' && num2 != 0)
      return num1 / num2;
    if(op == '*')
      return num1 * num2;
    throw new ArgumentException();
  }
}